### DISCUSSION TITLE
Conditional powerline symbol

### DISCUSSION THREAD

**Discussion by maru-sama** [JanDeDobbeleer/oh-my-posh#3847] [Ideas]
I looked through the docs and I do not think this is possible right now. My goal is  to  have a round powerline symbol for the last segment, while having a > for the others. Since my last segment is "git" it is not always shown this is a problem.

My idea is to have a conditional like, if last segment then x else y. Would this be possible?

**Comment by maru-sama** [comment]
This of course would also work in the template, which I think would make more sense, then doing this on the powerline symbol

**Comment by JanDeDobbeleer** [comment]
@maru-sama you need to have a look at the combination of a diamond segments without a trailing diamond and an ending text segment that uses the parentBackground as foreground with the symbol you aim to use last.

**Reply by maru-sama** [reply]
That's what I tried as welll but did not succeed, What I want is

No git:
segment1>segment2)

Git:
segment1>segment2>gitsegment)

If I remove the trailing diamond from segment2 it ends with a | in the no git case. I am pretty sure I am missing something here...

**Reply by JanDeDobbeleer** [reply]
if you can share that config I'll gladly have a quick look for you. This is perfectly possible I think.

**Comment by maru-sama** [comment]
These are the two segments I am trying it with, the first one is a simple date, the second nad last one the git status. The goal as I said is to have a round end on the last segment.

[maru.omp.txt](https://github.com/JanDeDobbeleer/oh-my-posh/files/11478406/maru.omp.txt)

**Reply by maru-sama** [reply]
sorry the code block got screwed, so I attached my complete theme file

**Comment by maru-sama** [comment]
Ok, I fixed this. On the second to last segment I used the following template
`"<parentBackground></> {{ .CurrentDate | date .Format }} {{ if not (.Segments.Contains \"Git\") }}<background,transparent></>{{end}}"
`
and on the git segment I did something similar. This works nicely.

Thanks for pointing me in the right direction
